hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -168,12 +168,37 @@ registers: [
       },
     },
 
+    { multireg: {
+        cname: "SENSOR_CTRL",
+        name: "ALERT_EN",
```

Comment:
@moidx @timothytrippel do you think it is ok to handle them as "opt in"?

Any ROM changes needed for this?

Reply:
Hi @msfschaffner, I added https://github.com/lowRISC/opentitan/issues/21871 to track changes for the ROM.